src/Bicep.LangServer/Snippets/Snippet.cs

```diff
@@ -14,11 +14,11 @@ namespace Bicep.LanguageServer.Snippets
     public sealed class Snippet
     {
         // Below regex is used to detect following snippet syntax:
-        // ${(?<index>\d+):(?<name>\w+)} detects placeholders i.e. tab stops with values e.g ${1:foo}
+        // ${(?<index>\d+):(?<name>\w+|\*|(\w+\/\w+)+@(\w+|(\d{4}-\d{2}-\d{2})))} detects placeholders i.e. tab stops with values e.g ${1:foo}
         // $(?<index>\d+) detects tab stops e.g. $1
         // $(?<index>\d+)\|((?<name>[^,]+)(?<value>.*))\|} detects placeholders with choices e.g. ${1|one,two,three|}
```

Comment:
In this example (`${1|one,two,three|}`), it looks like the regex here is going to give you the following capture groups:
```
index: 1
name: one
value: ,two,three
```

Is that expected?

Reply:
Yes, that's correct. Removed <value> here:
https://github.com/Azure/bicep/commit/1613a076e3fa7e8d6faac75cfdf416a09c037924

We'll use the first one by default. Let me know if you have any concerns.

Reply:
Thanks, makes sense!